src/components/30-organisms/table-sortable/react/demo.jsx

```diff
@@ -20,7 +20,7 @@ const model = {
       { html: '<span>55</span>' },
       { html: '<span>Peter</span>' },
       { html: '<span>Winterthur</span>' },
-      { html: '<span>22.04.2019</span>' },
+      { text: '<span>22.04.2019</span>' },
```

Comment:
It is not possible to sort this column. If you click on the header nothing happens. Pls have a look why.